CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.99.0]
```

Comment:
How about something in the changelog noting that this is a pre-release line for the 1.0 release.